src/account_multisig.ts

```diff
@@ -157,7 +216,34 @@ export class AccountMultisig extends Account {
         return requestId
     }
 
-    async verifyRequestCode(securityCode: string) {
+    async getCodeDefault(): Promise<string> {
+        if (typeof window === 'undefined') {
+            throw new Error('There is no getCodeFallback for Node env. Please provide your own in AccountMultisig constructor options argument.');
+        }
+        const method = await this.get2faMethod();
+        if (!method || !method.kind) {
+            throw new Error('no active 2fa method found');
+        }
+        const code = window.prompt(`Enter security code sent to ${ method.kind.split('2fa-')[1] }`)
```

Comment:
let's not introduce dependency on browser here. I think it's fine if default is just throwing an error if callback is not set